src/main/java/duke/common/Utils.java

```diff
@@ -0,0 +1,105 @@
+package duke.common;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The Utils class provides various utility methods for working with date, time, and integer values.
+ */
+public class Utils {
+    private static final String FORMAT = "d/M/yyyy HHmm";
+
+    /**
+     * Checks if a given string represents a valid date and time in a specific format.
+     * 
+     * @param dateTime The dateTime parameter is a string representing a date and time in a specific
+     *                  format.
+     * @return The method is returning a boolean value. It returns true if the given dateTime string is
+     *         a valid date and time according to the specified format, and false otherwise.
+     */
+    public static boolean isValidDateTime(String dateTime) {
+        dateTime = dateTime.trim();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(FORMAT);
+        try {
+            LocalDateTime.parse(dateTime, formatter);
+        } catch (DateTimeParseException e) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * The function checks if two date-time strings are valid and if the "from" date-time is before or
+     * equal to the "to" date-time.
+     * 
+     * @param fromDateTime The starting date and time.
+     * @param toDateTime The ending date and time.
+     * @return The method is returning a boolean value. It returns true if the fromDateTime is before or
+     *         equal to the toDateTime, and false otherwise. If either of the date strings cannot be
+     *         parsed into LocalDateTime objects, it returns false.
+     */
+    public static boolean isValidDateTime(String fromDateTime, String toDateTime) {
+        fromDateTime = fromDateTime.trim();
```

Comment:
Maybe the naming of this method will cause some confusion as it has the same name as the previous method but they are checking a different thing. Maybe can rename the previous one to isValidDateTimeFormat?